File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int sum = 0;
        int result = 0;

        for(int i = 0; i<nums.size(); i++) {
            sum += nums[i];
        }
        for(int i = 0; i<nums.size()+1; i++) {
            result += i;
        }

        return result - sum;
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n = nums.length;
        int sum  = 0;
        int res = 0;
        for(int i = 0; i<n ;i++) {
            sum += nums[i];
            res += i;
        }
        res += nums.length;
        return res - sum;

        
    }
}